undefined reference to `pjmedia_conf_set_op_cb' when PJMEDIA_CONF_USE_SWITCH_BOARD is set

### Describe the bug

when compiling with #define PJMEDIA_CONF_USE_SWITCH_BOARD 1 
in config_site.h
linking fails with:
**undefined reference to `pjmedia_conf_set_op_cb'** 
in function `pjsua_aud_subsys_init'
in function `pjsua_conf_set_op_cb'


### Steps to reproduce

check out pjmedia master 72f4c00b
write 
#define PJMEDIA_CONF_USE_SWITCH_BOARD 1 
in pjlib/include/pj/config_site.h
./configure
make

### PJSIP version

current master 72f4c00b
The bug was introduced in commit 4d1b0f8791155131bd5cef732b21431c73bbf7a9

### Context

Linux MINT 21.3

config_site.h only:
#define PJMEDIA_CONF_USE_SWITCH_BOARD 1 

./configure
make

### Log, call stack, etc

```shell
not relevant, compile fails
```

--- a/pjmedia/src/pjmedia/conf_switch.c
+++ b/pjmedia/src/pjmedia/conf_switch.c
@@ -297,7 +297,7 @@ static pj_status_t resume_sound( pjmedia_conf *conf )
 }
 
 
-/**
+/*
  * Destroy conference bridge.
  */
 PJ_DEF(pj_status_t) pjmedia_conf_destroy( pjmedia_conf *conf )
@@ -308,6 +308,20 @@ PJ_DEF(pj_status_t) pjmedia_conf_destroy( pjmedia_conf *conf )
     pj_mutex_destroy(conf->mutex);
 
     return PJ_SUCCESS;
+}
+
+
+/*
+ * Register the callback to be called when a port operation has been
+ * completed.
+ */
+PJ_DEF(pj_status_t) pjmedia_conf_set_op_cb(pjmedia_conf *conf,
+                                           pjmedia_conf_op_cb cb)
+{
+    PJ_UNUSED_ARG(conf);
+    PJ_UNUSED_ARG(cb);
+
+    return PJ_ENOTSUP;
 }
 
 

